aten/src/ATen/native/quantized/library.cpp

```diff
@@ -1,13 +1,5 @@
 #include <torch/library.h>
-
-int register_linear_params();
-
-template <int kSpatialDim = 2>
-int register_conv_params();
-
-extern template int register_conv_params<2>();
-extern template int register_conv_params<3>();
-int register_embedding_params();
+#include <aten/src/ATen/native/quantized/cpu/fbgemm_utils.h>
```

Comment:
This type of change is bad, as it add dependency between native and native/cpu. We probably should move fbgemm_utils.h to native if we want to do something like that

Reply:
Follow the suggestion by moving the declarations to aten/src/ATen/native/quantized/library.h